resources/i18n/pt-br.json

```diff
@@ -0,0 +1,38 @@
+{
+	"@metadata": {
+		"authors": [
+			"Eduardo Addad de Oliveira"
+		]
+	},
+	"error-downloader-launch-message": "Impossível de baixar, o Kiwix-desktop será iniciado, mas todas as funções não funcionarão!",
+	"open-zim": "Abrir Zim",
+	"print": "Imprimir",
+	"browse-library": "Navegar na biblioteca",
+	"open-recent": "Aberto recentemente",
+	"reading-list-title": "Lista de leitura",
+	"zoom-reset": "Redefinição de zoom",
+	"help": "Ajuda",
+	"feedback": "Comentário",
+	"report-a-bug": "Relatar um erro",
+	"tools": "Ferramentas",
+	"back": "Voltar",
+	"search-files": "Pesquisar arquivos",
+	"title": "Título",
+	"resume": "Retomar",
+	"pause": "Pausar",
+	"cancel": "Cancelar",
+	"apply": "Aplicar",
+	"reset": "Redefinir",
+	"version": "Versão",
+	"build-date": "Data de construção",
+	"report-issue-3/3": "Mencione a versão e a data de compilação no problema.",
```

Comment:
The `report-issue` translation is splitted into 3 parts, they all need to be translated otherwise there will be only a part of the sentence will be translated, the other parts will be in english.

Reply:
@jetownfeve21 Partial translation is a general problem. The problem your report just make it a bit more accute... I see no solution to that problem @Abijeet Do you?

Reply:
Sorry, I haven't had a chance to look at this yet. I'll get back to you tomorrow.

Reply:
Why is the message split into three parts in the first place?